### PR TITLE
Enable more linters in CI.

### PR DESCRIPTION
**Pull request by atombrella**
This enables the last four linters left out from #438 Tests currently fail, and I'll try to fix them today. See #440 for reference.

While doing this, I had a look at `go.mod` Are you sure 1.13 will work with the current code?

### CODE REVIEW

File: cmd/kube-score/main.go [hunk 1]
```diff
@@ -242,7 +242,10 @@ func listChecks(binName string, args []string) {
 	fs := flag.NewFlagSet(binName, flag.ExitOnError)
 	printHelp := fs.Bool("help", false, "Print help")
 	setDefault(fs, binName, "list", false)
-	fs.Parse(args)
+	err := fs.Parse(args)
+	if err != nil {
+		panic(err)
+	}
```
Comment (atombrella) [inline cmd/kube-score/main.go:248]: I used this pattern in all places where I fixed the warnings from `errcheck`. Most likely there are fancier ways to do this; however, I saw some use of this in other places of the code. I suppose it's equivalent in many ways to `try/except: raise` in Python (and other OOP languages that have a `try/catch` statement).

Comment (zegl) [reply cmd/kube-score/main.go:248]: Nice fixes!

Generally, I think that it's a good practice to only allow panics in the `main()` function. Other functions should instead return an error if they fail.

So for the `listChecks` function, modify it to return an error, and change this to a `if err != nil { ... }`.

**Review by zegl — APPROVED** [review]
Looks good to me! ⭐ 

bors merge

**Comment by atombrella** [comment]
@zegl I think you need to write it `bors merge` in a comment with only that line?

**Comment by zegl** [comment]
Huh, it's unclear why this was not merged. Lets try again!

bors merge

**Pull request merged**